tools/mage/go.go

```diff
@@ -71,6 +71,10 @@ func outputGo(cmd string, args ...string) (string, error) {
 	return buf.String(), nil
 }
 
+func runGoMute(args ...string) error {
```

Comment:
Instead change `runGo` to have the same behavior as `mage`: https://github.com/magefile/mage/blob/310e198ebd9303cd2c876d96e79de954915f60a7/sh/cmd.go#L64-L69